latch_cli/types.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+from dataclasses import dataclass
+
+from dataclasses_json import dataclass_json
+
+
+@dataclass_json
```

Comment:
`dataclass_json` is an awful library and should arguably not be used by anyone

just do `LatchWorkflowConfig(**json.load(f))` instead